gui-agent/vmside.c

```diff
@@ -52,7 +52,7 @@
 #include "encoding.h"
 #include <libvchan.h>
 
-#define SOCKET_ADDRESS  "/var/run/xf86-qubes-socket"
+#define SOCKET_ADDRESS  "/run/xf86-qubes-socket"
```

Comment:
For the above reason, I'd keep this and the other place for this path referencing /var/run.